test/reconciler.test.js

```diff
@@ -218,8 +231,8 @@ describe('reconciler', () => {
 
       incrementGeneration()
       wrapper.setProps({ second: false })
-      expect(First.rendered).toHaveBeenCalledTimes(5)
-      expect(Second.rendered).toHaveBeenCalledTimes(4)
+      expect(First.rendered).toHaveBeenCalledTimes(6)
```

Comment:
This is "the main concern" -  I have no idea why this is happing, no explanation, nothing...